src/darker/import_sorting.py

```diff
@@ -1,29 +1,27 @@
 import logging
-from pathlib import Path
-from typing import List, cast
 
 try:
-    from isort import SortImports
+    import isort
 except ImportError:
-    SortImports = None
+    isort = False
 
 logger = logging.getLogger(__name__)
 
 
 def apply_isort(content: str) -> str:
-    logger.debug(
-        "SortImports(file_contents=..., check=True, multi_line_output=3, "
-        "include_trailing_comma=True, force_grid_wrap=0, use_parentheses=True, "
-        "line_length=88, quiet=True)"
-    )
-    result = SortImports(
-        file_contents=content,
-        check=True,
+    isort.check_code(code=content)
```

Comment:
Why is this call needed?

Reply:
This replaces `check=True` and writes error messages to `stdout`. See changes in `test_main`.

Reply:
Do we need the error messages?

And if we do, is there a cleaner way to capture them than `stdout`?

Reply:
> Do we need the error messages?

I don't know... Some tests depend on error messages. Probably, we could raise an exception if `check_code` is `False`

> And if we do, is there a cleaner way to capture them than `stdout`?

Hm... `isort` has hardcoded prints to write to `stdout`

Reply:
Well, I had to include those error messages in the test assertions, because in `isort<5` there was no way to get rid of the messages.

Now that we can run `isort` without polluting `stdout` with error messages, I think we should do that for simplicity's sake, and fix the tests as well – unless we get some benefit from having those error messages.

Reply:
I thought `check=False` in `isort<5` was for this?
I vote to get rid of error messages in another PR in order to keep this PR clear.

Reply:
> I vote to get rid of error messages in another PR in order to keep this PR clear.

I created another PR: #17